gazelle/sass.go

```diff
@@ -0,0 +1,165 @@
+/* Copyright 2019 The Bazel Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// gazelle provides a minimal implementation of language.Language for
+// rules_sass.
+package gazelle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/bazelbuild/bazel-gazelle/config"
+	"github.com/bazelbuild/bazel-gazelle/language"
+	"github.com/bazelbuild/bazel-gazelle/rule"
+)
+
+var _ = fmt.Printf
+
+type sasslang struct{}
+
+// NewLanguage returns an instace of the Gazelle plugin for rules_sass.
+func NewLanguage() language.Language {
+	return &sasslang{}
+}
+
+// Kinds returns a map of maps rule names (kinds) and information on how to
+// match and merge attributes that may be found in rules of those kinds. All
+// kinds of rules generated for this language may be found here.
+func (s *sasslang) Kinds() map[string]rule.KindInfo {
+	return map[string]rule.KindInfo{
+		"sass_library": {
+			MatchAny: true,
```

Comment:
`MatchAny` means that when merging a `sass_library`, the merger can match any target with the kind `sass_library`, regardless of name. This probably doesn't make sense if there may be multiple `sass_library` targets in the same build file.

Reply:
Done.